README.md

```diff
@@ -1 +1,20 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] README file should be descriptive, please use this [template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Reply:
I updated the README accordingly. Thank you for the feedback.